ert/data/record/_record.py

```diff
@@ -241,27 +319,53 @@ def _flatten_record_dict(
 
     @property
     def data(self) -> record_data:
-        # internally, recordtree is represented as BlobRecord; we don't need data here
+        """Internally, RecordTree is represented as BlobRecord; therefore
+        this property shoudn't be used.
+
+        Returns:
+            record_data: no data
+        """
+        #
         return json.dumps("RECORD_TREE").encode("utf-8")
 
     @property
     def record_type(self) -> RecordType:
+        """Returns either :class:`RecordType.BLOB_TREE` or
+        :class:`RecordType.NUMERICAL_TREE`
+
+        Returns:
+            RecordType: RecordType.BLOB_TREE or RecordType.NUMERICAL_TREE
+        """
         return RecordType.BYTES
 
     @property
     def flat_record_dict(self) -> Dict[str, RecordGen]:
+        """Returns a flattened dictionary, where each `key` represents a path
+        and `value` represents the actual Record object.
```

Comment:
"The `:` character is used in the keys to identify the branches in the original tree."

Reply:
Actually is `/` but it is still good to mention it.